### DISCUSSION TITLE
アドベントカレンダーのリマインド通知に記載されている日付が1日ずれている

### DISCUSSION THREAD

**Discussion by fussy113** [increments/qiita-discussions#586] [要望・アイデア・不具合]
通知一覧のページを見て気がついたので、こちら投稿します:bow:

<img width="800" alt="通知ページ" src="https://github.com/increments/qiita-discussions/assets/15154105/7e7b5a02-8d43-4ef9-8922-1abf8ff11c6e">

`2023/12/10は「Vue」アドベントカレンダーのあなたの担当日です。` と通知には記載あったのですが、[Vueのアドベントカレンダーページ](https://qiita.com/advent-calendar/2023/vue)をみると、自分の担当は`12/11`です。
(12/10はすでに別の方がカレンダーを入れている、自分が後から投稿の日付を変更したなどは無かったです)

<img width="175" alt="アドベントカレンダー詳細ページのカレンダー部分" src="https://github.com/increments/qiita-discussions/assets/15154105/f380a7f4-d4a9-476f-aa15-90b8abaa0ad3">

`2023/12/11は「カレンダー名」アドベントカレンダーのあなたの担当日です。` となるのが正しいかと思いました。

ご確認をお願いいたします:bow:

**Comment by tanaka-asami415** [comment]
@fussy113 
ご報告ありがとうございます！
不具合のため修正します🙇（今年は間に合わなさそうなので、来年になりそうです🙏）

**Reply by P-SiZK** [reply]
@fussy113 
大変お待たせしました :bow:
リマインド通知の日付がずれてしまう不具合の修正を行いました！
